### ISSUE TITLE
Update legislation page for new Admin Fine extension law.

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2638]
**User story** 
As a filer (or a stakeholder or a consultant), I need to know that there are automatic penalties for late/non-filed reports, and that the law for those penalties has been extended.

**Background:**
Public Law No: 115-386 (12/21/2018) amended FECA to extend through 2023 the administrative penalty authority of the Federal Election Commission

**To do**
- [x] Update the legislation page to add this law. (Moved this task out of #2627)
- [x] Ensure admin fine pages are updated for the new expiration date.

**Comment by dorothyyeager** [comment]
GPO has not yet posted the Pub. Law. on its site.

Congress.gov has the enrolled version: https://www.congress.gov/115/bills/hr7120/BILLS-115hr7120enr.pdf. This URL can be used until we get the GPO link

**Comment by dorothyyeager** [comment]
I've updated https://www.fec.gov/legal-resources/legislation/ to add the new legislation. (We'll need to remember to update the link to the legislation's text when govinfo publishes the public law, but for now we've got a link to the enrolled bill as on congress.gov.

I've also reviewed the admin fine pages and they all reflect the latest version of this part of the law.

Closing as work for now is complete.
